### PR TITLE
Creates the Syndicate Beer and places it in the uplink under Badassery

### PR DESCRIPTION
**Pull request by aguyiguess**
## About The Pull Request

adds syndicate beer to drinks.dm, adds the icon, and places it in the uplink under Badassery for 4 TC. It contains 10u of Beer and 20u of Anti-Toxin.
## Why It's Good For The Game

It provides a quick way to get rid of poisons and toxin damage if you can't go to medbay. Essentially syndicate smokes but for toxin damage.

## Changelog
:cl:
add: Added the Syndicate Beer to the uplink under "Pointless Badassery" for 4 TC.
/:cl:

### CODE REVIEW

**Comment by aguyiguess** [comment]
hrnnng no review

**Review by PowerfulBacon — DISMISSED** [review]

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -265,6 +265,13 @@
 	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
 	foodtype = GRAIN | ALCOHOL
 
+/obj/item/reagent_containers/food/drinks/syndicatebeer
+	name = "suspicous looking beer"
+	desc = "The bottle is painted black with an S on the label. Looks like some sort of enemy corporation."
```
Comment (Crossedfall) [inline code/modules/food_and_drinks/drinks/drinks.dm]:
```suggestion
	desc = "The bottle is painted black with an S on the label. Looks like some sort of enemy corporation."
```

I can't entirely explain why, but I don't really like how the last bit is worded, "Looks like some sort of enemy corporation." It's common in-universe knowledge who the Syndicate Organization is, not sure if we need to explain it here as well.

This is an overall really minor gripe though

Comment (Crossedfall) [reply code/modules/food_and_drinks/drinks/drinks.dm]: Actually, I just realized, Syndicate players are going to be buying this right? Wouldn't they know who they work for? Also would they label their own organization as an "enemy corporation?"

Comment (aguyiguess) [reply code/modules/food_and_drinks/drinks/drinks.dm]: i didnt think about that, good point

**Comment by PowerfulBacon** [comment]
A round not consume more than 1 in quick succession label should be on the beer itself too.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**